packages/vector_graphics_compiler/test/parser_test.dart

```diff
@@ -6,6 +6,24 @@ import 'package:vector_graphics_compiler/vector_graphics_compiler.dart';
 import 'test_svg_strings.dart';
 
 void main() {
+  test('text with transform', () async {
+    final VectorInstructions instructions = await parse(
+      '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 160 160"><text transform="translate(10, 11)">a</text></svg>',
```

Comment:
I think it would be safe to take translations and encode them in the baseline start. We should only need to encode scale/skew/rotations at runtime.

Reply:
Changed to `rotate(10)`